libs/authorizenet/src/drivers/interfaces/authorize-net-config.interface.ts

```diff
@@ -1,4 +1,9 @@
+import { InjectionToken } from '@angular/core';
+
+export const DaffAuthorizeNetConfigToken = new InjectionToken('DaffAuthorizeNetConfigToken');
+
 export interface DaffAuthorizeNetConfig {
 	clientKey: string;
 	apiLoginID: string;
+	acceptJsUrl: string;
```

Comment:
I think instead of forcing the end-user to lookup the URL, we should pre-provide the prod/staging URIs and switch on a flag.

Reply:
Is it guaranteed to always be the same relative to the backend's base URL?

Reply:
@griest024 yea

@damienwebdev oops yea we talked about that

Reply:
done